cmd/erasure-server-pool-decom.go

```diff
@@ -1327,11 +1327,7 @@ func (z *erasureServerPools) Status(ctx context.Context, idx int) (PoolStatus, e
 	poolInfo := z.poolMeta.Pools[idx].Clone()
 	if poolInfo.Decommission != nil {
 		poolInfo.Decommission.TotalSize = pi.Total
-		if poolInfo.Decommission.Failed || poolInfo.Decommission.Canceled {
-			poolInfo.Decommission.CurrentSize = pi.Free
-		} else {
-			poolInfo.Decommission.CurrentSize = poolInfo.Decommission.StartSize + poolInfo.Decommission.BytesDone
```

Comment:
this was done for accuracy reasons, what is the problem here?

Reply:
well, I am not able to understand the logic here.. CurrentSize starts with the free raw space of drives then we add the object sizes that were decommissioned

Reply:
StartSize is the first time what was the current Free @vadmeste when the decom was started it should never skew in the manner it skewed for the customer. 

So we keep adding bytesDone to ensure that we keep the Current free size based on the original startSize.

Reply:
Okay understood, CurrentSize is close to pi.Free then. 

Something about poolInfo.Decommission.StartSize is that it is a persistant value and it depends on the available number of drives when StartSize was calculated the first time. In my opinion, we could use realtime values instead since we are doing that few lines with z.getDecommissionPoolSpaceInfo(idx)

Reply:
yeah if that is the case this entire code is a bit meaningless you just need 

```
		poolInfo.Decommission = &PoolDecommissionInfo{
			TotalSize:   pi.Total,
			CurrentSize: pi.Free,
		}
```

Reply:
yes, but this is Status() function, we cannot initialize &PoolDecommissionInfo if there is a an ongoing decommissioning, we still want to return other information such as StartTime, etc..